test/ComparisonExtensionsTests/DoubleComparisonExtensionsTest.cs

```diff
@@ -0,0 +1,310 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using ComparisonExtensions;
+using FluentAssertions;
+using Xunit;
+
+namespace ComparisonExtensionsTests
+{
+    public class DoubleComparisonExtensionsTest
+    {
+        public record DoubleEqualityScenario(double Left, double Right, string Name = null)
+        {
+            public override string ToString() => this.Name == null
```

Comment:
We can use the null coalescing expression here. There are more of these. Please, do this for all of them.

Suggested code:
```CSharp
public override string ToString() => this.Name ?? $"left = {this.Left}, right = {this.Right}";
```

Reply:
updated